releases/7.6.0.md

```diff
@@ -29,65 +29,66 @@ Arches 7.6.0 Release Notes
-- 10121 Prevent language code duplication during concept import; avoid creating Language objects other than English in new projects
-- 10575 Add pre-commit hooks for formatting
-- 10781 Graph.delete_instances() now uses BulkDataDeletion
-- 10665 Return resourceid in ActivityStream rather than resource url
-- 11009 Move arches application declaration to AppConfig
-- 10754 Move the i18n/ url outside of the i18n_patterns function
-- Fixes hard coded Polygon geometry type in geo_utils.convert_multipart_to_singlepart #11218
-- Fixes geocoder placeholder #11173
-- Fixes error preventing user from changing a node's widget #10899
-- 11274 Raise limit of editable installs of arches applications from 3 to 9
-- Allow minzoom and maxzoom to be applied to a resource map source from a geojson node config #10929
+- Update Google Analytics syntax supporting Google Analytics 4 [#11132](https://github.com/archesproject/arches/issues/11132)
+- Filter out tiles created during resource creation from activity stream API [#9768](https://github.com/archesproject/arches/issues/9768)
+- Ensure resource creation edit log timestamps precede resource update timestamps [#9769](https://github.com/archesproject/arches/issues/9769)
+- Support Shapefile import in map editor widget [#11339](https://github.com/archesproject/arches/issues/11339)
+- Fix nuisance failing requests to help template view [#11478](https://github.com/archesproject/arches/issues/11478)
+- Adds Github action for comparing test coverage between branches and rejecting branches that lower test coverage [#10738](https://github.com/archesproject/arches/issues/10738)
+- Ensure search facets have widget label on advanced search restore [#11046](https://github.com/archesproject/arches/issues/11046)
+- Persist unsaved value in string widget if a user switches between cards in the card tree [#11092 ](https://github.com/archesproject/arches/issues/1092 )
+- Fix workbench sidepanel header misalignment occuring during provisional edit review [#11323](https://github.com/archesproject/arches/issues/11323)
+- Add project-level testing and GitHub test runners [#10842](https://github.com/archesproject/arches/issues/10842)
+- Lenient file-type checking mode [#10862](https://github.com/archesproject/arches/issues/10862)
+- Allow overriding search_results view [#10699](https://github.com/archesproject/arches/issues/10699)
+- Styling fix in resource model manage menu [#10911](https://github.com/archesproject/arches/issues/10911)
+- Harden SystemSettings model against too-early access [#11118 [#11118]](https://github.com/archesproject/arches/issues/1118])
+- Upgrade openpyxl package to 3.1.2 and fixes ETL modules [#10726](https://github.com/archesproject/arches/issues/10726)
+- Implement arches version compatibility check as a Django system check [#11114](https://github.com/archesproject/arches/issues/11114)
+- Add unlocalized string datatype [#9191](https://github.com/archesproject/arches/issues/9191)
```

Comment:
I might promote this one up to the highlights.

Reply:
Agreed